###
### THIS FILE IS AUTOMATICALLY GENERATED by ./marvin_controller_configuration_gen.sh /home/n130s/link/ROS/indigo_trusty/cws_marvin/src/tork-a/marvin/marvin_controller/config Wed Apr 29 12:45:40 JST 2015
###

joint_state_publisher:
   controller:
       package: marvin_controller
       module: joint_state_publisher
       type: JointStatePublisher


# fr_rotation_joint
fr_rotation_joint_position_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: fr_rotation_joint
    joint_speed: 2.0
    motor:
        id: 1
        init: 2048
        min: 0
        max: 4095

# br_rotation_joint
br_rotation_joint_position_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: br_rotation_joint
    joint_speed: 2.0
    motor:
        id: 2
        init: 2048
        min: 0
        max: 4095

# bl_rotation_joint
bl_rotation_joint_position_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: bl_rotation_joint
    joint_speed: 2.0
    motor:
        id: 3
        init: 2048
        min: 0
        max: 4095

# fl_rotation_joint
fl_rotation_joint_position_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: fl_rotation_joint
    joint_speed: 2.0
    motor:
        id: 4
        init: 2048
        min: 0
        max: 4095

# fr_wheel_joint
fr_wheel_joint_velocity_controller:
    controller:
        package: dynamixel_controllers
        module: joint_torque_controller
        type: JointTorqueController
    joint_name: fr_wheel_joint
    joint_speed: 2.0
    motor:
        id: 5
        init: 0
        min: -1024
        max: 1024

# br_wheel_joint
br_wheel_joint_velocity_controller:
    controller:
        package: dynamixel_controllers
        module: joint_torque_controller
        type: JointTorqueController
    joint_name: br_wheel_joint
    joint_speed: 2.0
    motor:
        id: 6
        init: 0
        min: -1024
        max: 1024

# bl_wheel_joint
bl_wheel_joint_velocity_controller:
    controller:
        package: dynamixel_controllers
        module: joint_torque_controller
        type: JointTorqueController
    joint_name: bl_wheel_joint
    joint_speed: 2.0
    motor:
        id: 7
        init: 0
        min: -1024
        max: 1024

# fl_wheel_joint
fl_wheel_joint_velocity_controller:
    controller:
        package: dynamixel_controllers
        module: joint_torque_controller
        type: JointTorqueController
    joint_name: fl_wheel_joint
    joint_speed: 2.0
    motor:
        id: 8
        init: 0
        min: -1024
        max: 1024
